org.eclipse.jdt.core/dom/org/eclipse/jdt/core/dom/TypePattern.java

```diff
@@ -187,10 +187,10 @@ final ASTNode internalGetSetChildProperty(ChildPropertyDescriptor property, bool
 	 * </ul>
 	 * @exception UnsupportedOperationException if this operation is used other than JLS19
 	 * @exception UnsupportedOperationException if this expression is used with previewEnabled flag as false
-	 * @Deprecated(forRemoval = true, since="2024-07")
-	 * In the JLS22 API, this method is replaced by  <code>setPatternVariable(VariableDeclaration patternVariable)</code>
+	 * @deprecated In the JLS22 API, this method is replaced by  <code>setPatternVariable(VariableDeclaration patternVariable)</code>
```

Comment:
If you format this as described here:

https://eclipse.dev/eclipse/news/4.30/jdt.html#deprecated-cleanup

users should even get a quickfix!

Reply:
Thanks for this information. I have pushed the code. Could you please re-review the PR.